File: Turquoise/UI/Common/UIView+NibLoader.m
#import "UIView+NibLoader.h"

@implementation UIView (NibLoader)

+ (UIView *)tq_loadFromNib:(NSString *)nibName owner:(id)owner {
  if (nibName.length == 0) {
    return nil;
  }
  UIView *view = [[NSBundle mainBundle] loadNibNamed:nibName owner:owner options:nil].firstObject;
  return view;
}

@end

File: Turquoise/UI/Common/UIView+NibLoader.swift
import Foundation
import UIKit

extension UIView {
  class func tq_load(from nibName: String, owner: Any) -> UIView? {
    let objects = Bundle.main.loadNibNamed(nibName, owner: owner, options: nil)
    return objects?[0] as? UIView
  }
}
